cloud/passportauth/example/index.ts

```diff
@@ -32,36 +36,75 @@ app.use(bodyParser.urlencoded({
   extended: false
 }));
 app.use(cors());
-authService.init(app, sessionOpts);
 
-app.get('/testAdminEndpoint', authService.protect('admin'), (req: express.Request, res: express.Response) => {
-  res.json({ routeName: '/testAdminEndpoint', msg: 'authorized to access admin endpoint' });
+app.get('/', (req: express.Request, res: express.Response) => {
+  res.json({ name: 'Passport Auth Module Example' });
 });
 
-app.get('/testUserEndpoint', authService.protect('user'), (req: express.Request, res: express.Response) => {
-  res.json({ routeName: '/testUserEndpoint', msg: 'authorized to access user route' });
-});
+// Setting up authService using session based authentication
+const sessionRouter = express.Router();
+sessionAuthService.init(sessionRouter, sessionOpts);
 
-app.get('/', (req: express.Request, res: express.Response) => {
-  const payload = { msg: 'default route', user: req.user, session: req.session};
-  res.json(payload);
+sessionRouter.get('/session-login', (req: express.Request, res: express.Response) => {
+  res.sendFile(path.join(__dirname, 'public/session-login.html'));
 });
 
-app.get('/testEndpoint', authService.protect(), (req: express.Request, res: express.Response) => {
-  res.json({ routeName: '/testEndpoint', msg: 'user is authenticated, no role required for this resource' });
+sessionRouter.post('/session-login', sessionAuthService.authenticate('local', {
+  successReturnToOrRedirect: '/testSessionEndpoint'
+}));
+
+sessionRouter.get('/testSessionAdminEndpoint', sessionAuthService.protect('admin'),
+  (req: express.Request, res: express.Response) => {
+  res.json({ routeName: '/testSessionAdminEndpoint', authType: 'session', msg: 'authorized to access admin endpoint' });
 });
 
-app.get('/login', (req: express.Request, res: express.Response) => {
-  res.sendFile(path.join(__dirname, 'public/login.html'));
+sessionRouter.get('/testSessionUserEndpoint', sessionAuthService.protect('user'),
+  (req: express.Request, res: express.Response) => {
+  res.json({ routeName: '/testSessionUserEndpoint', authType: 'session', msg: 'authorized to access user endpoint'});
 });
 
-app.post('/login', authService.authenticate('/'));
+sessionRouter.get('/testSessionEndpoint', sessionAuthService.protect(),
+  (req: express.Request, res: express.Response) => {
+  res.json({ routeName: '/testSessionEndpoint', authType: 'session',
+             msg: 'authorize to access endpoint, no roles required'});
+});
 
-app.get('/logout', (req: express.Request, res: express.Response) => {
+sessionRouter.get('/logout', (req: express.Request, res: express.Response) => {
   req.logout();
   res.redirect('/');
 });
 
+// Setting up authService using token based authentication
+const tokenRouter = express.Router();
+const secret = 'demo_secret';
+tokenAuthService.init(tokenRouter, undefined, secret);
+
+tokenRouter.get('/token-login', (req: express.Request, res: express.Response) => {
```

Comment:
Missing comments. There are two ways for that. 
You can have two servers one for cookie one for token.
Or just mention that this endpoints are going to work with JWT etc.